x-pack/plugins/upgrade_assistant/__jest__/client_integration/overview/fix_logs_step/fix_logs_step.test.tsx

```diff
@@ -7,6 +7,17 @@
 
 import { act } from 'react-dom/test-utils';
 
+const MOCKED_TIME = '2021-09-05T10:49:01.805Z';
```

Comment:
Once the logs team register the kibana locators in their app, we should be able to remove this mock and follow a similar approach to how discover link is tested.

Reply:
Added a comment about it!